eth2_testgen/shuffling/core_helpers.py

```diff
@@ -8,26 +8,29 @@
 from eth_typing import Hash32
 
 from constants import EPOCH_LENGTH, SHARD_COUNT, TARGET_COMMITTEE_SIZE
-from enums import ValidatorStatusCode
 from utils import hash
-from yaml_objects import ShardCommittee, ValidatorRecord
+from yaml_objects import Validator
 
+EpochNumber = int
+ValidatorIndex = int
+Bytes32 = bytes
```

Comment:
These three should be `EpochNumber = NewType("EpochNumber", int)` etc.

Reply:
Done